pyrtl/rtllib/pyrtlfloat/floatoperations.py

```diff
@@ -2,14 +2,14 @@
 
 from ._add_sub import AddSubHelper
 from ._multiplication import MultiplicationHelper
-from ._types import PyrtlFloatConfig, RoundingMode
+from ._types import FloatingPointType, PyrtlFloatConfig, RoundingMode
 
 
 class FloatOperations:
     default_rounding_mode = RoundingMode.RNE
 
     @staticmethod
-    def multiply(
+    def mul(
```

Comment:
This doesn't seem very useful, since it just passes all its arguments through to `MultiplicationHelper.multiply`? Seems better to point users directly to `MultiplicationHelper.multiply`. Same comments for the other methods below.

Reply:
The reason I have this `FloatOperations` class and separate `MultiplicationHelper` and `AddSubHelper` classes is because the user is supposed to use `FloatOperations`, `MultiplicationHelper` and `AddSubHelper` are internal helpers and the user is not supposed to use them. I created these helpers so I can separate the multiplication and addition/subtraction logic into separate files.